libs/components/popovers/testing/src/dropdown/harness/dropdown-harness.ts

```diff
@@ -25,28 +26,28 @@ export class SkyDropdownHarness extends SkyComponentHarness {
   }
 
   /**
-   * Clicks the dropdown button
+   * Clicks the dropdown button.
    */
   public async clickDropdownButton(): Promise<void> {
     (await this.#getDropdownButton()).click();
   }
 
   /**
-   * Gets the id of the dropdown menu the button controls
+   * Clicks out of the dropdown button. If `dismissOnBlur` property is set to false, then the dropdown menu does not close.
    */
-  public async getAriaControls(): Promise<string | null> {
-    return (await this.#getDropdownButton()).getAttribute('aria-controls');
+  public async clickOutOfDropdownMenu(): Promise<void> {
+    (await (await this.#getOverlay())?.host())?.click();
```

Comment:
This function was added as a way to test dismissOnBlur functionality

Reply:
If we move this to dropdown menu harness, then if all a consumer wants to do is test that their dropdown is opening and closing correctly, they would still have to create both a `dropdownHarness` and a `dropdownMenuHarness` in order to test that.

Reply:
Correct. The `clickOut` method really only makes sense from the menu's perspective.
```
await(await dropdownHarness.getDropdownMenu())?.clickOut();
```